client-evaluation/README.md

```diff
@@ -1,15 +1,29 @@
 # Evaluating clients
 
-## We don't do rewrites.
+## 100% client satisfaction
 
-#### Are we able to maintain the project after it's finished
+Maximum client satisfaction on project delivery should be our mail goal. As such, we should learn about the business circumstances of the client and anticipate any possible miscommunications or errors in the assignment. Instead of making something the client wants at the start, we should aim for something he needs in the end.
 
-We want to do that for limited time only. Amount of maintained projects is limited. Exception might be creating own product.
+Situations to be careful about as they might be a red light:
 
-#### Is the client willing to take over the project maintenance/development
+* **Rewrites** rarely lead to a happy client, as continuous development of existing solutions is usually the better way
+* **Many specific requirements about UI without proper user testing process** indicate flawed practice and should probably lead to a product workshop before even any development starts
+* **Critical importance of maintenance** is a sign that the project is very important for the client, but the client should know that it is effective to delegate repetitive and mundane tasks to hired employees or cheaper providers. Our team cannot internally handle expanding maintenance models, so creating automation and using services should be always part of the project.
 
-We're looking for clients willing to incorporate inhouse team. We want to talk with bussiness people.
+## Managing risk of the client
 
-#### Does the client understand iterative development
+Any software project is a risk and there is a chance that the project doesn't end up to the client satisfaction. There are two possible ways how to acceptably process such situation:
 
-#### Does the client accepts user testing
+1. Return money to the client - Make sure that the risk is included in the price and that we **can leave the project at any time** without additional costs for the client.
```

Comment:
I can't imagine any project we would be comfortable with money-back guarantee. This type of guarantee is suitable for products like books or our screen recording.

Reply:
After a bit of thinking about it, that's the reason why I didn't call it "money back guarantee", but rather "Return money to the client" (upfront payment) or "Don't charge for our work and stop cooperation" (payment after acceptation).

Reply:
I should probably change the text accordingly.